spec/base_spec_helper.rb

```diff
@@ -205,6 +205,11 @@ def handle_webmock_error(exception)
   # You can use `rspec -n` to run only failed specs.
   config.example_status_persistence_file_path = "tmp/rspec-status.txt"
 
+  # raise I18n exception handler
+  I18n.exception_handler = proc do |exception, *_|
+    raise exception.to_exception
+  end
```

Comment:
This is not touching the `config` and therefore it doesn't need to be in this block. Maybe move it above the Rspec.config block?

Reply:
Also, the earlier we declare this, the more code is covered. Any code that is loaded before these lines, won't raise exceptions. So I would move it as early as possible. Maybe after Webmock and before loading the support helpers?